omni_dior.mk

```diff
@@ -27,12 +27,12 @@ PRODUCT_COPY_FILES += \
     bionic/libc/zoneinfo/tzdata:recovery/root/system/usr/share/zoneinfo/tzdata
 
 PRODUCT_COPY_FILES += \
-    device/xiaomi/armani/kernel:kernel \
-    device/xiaomi/armani/dt.img:dt.img
```

Comment:
We don't need this, it was only a try to fix a build issue (dt.IMG not found) Now i Know why this didn't worked :D  So we can remove this lines